src/test/resources/cartridge/topology.lua

```diff
@@ -0,0 +1,15 @@
+cartridge = require('cartridge')
+replicasets = {{
+    alias = 'router',
+    roles = {'vshard-router', 'crud-router', 'app.roles.api_router'},
+    join_servers = {{uri = '0.0.0.0:3301'}}
+}, {
+   alias = 's1-storage',
```

Comment:
Looks odd that you have 3 spaces indented here instead of 4 spaces as above, maybe it's my code problem

Reply:
It's my typo